index.html

```diff
@@ -16,11 +16,15 @@
     <meta name="msapplication-TileColor" content="#da532c">
     <meta name="theme-color" content="#ffffff">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" type="text/css" href="baguetteBox/baguetteBox.css">
+    <script src="baguetteBox/baguetteBox.js"></script>
 <body>
 <div class='width-wrap'>
     <header id="top">
            <nav>
-            <label class="burger"></label><input type="checkbox" id="burger-toggle">
+              <input type="checkbox" id="burger-toggle">
+              <label class="burger" for="burger-toggle"></label>
```

Comment:
You need to put the checkbox first, then the label, so that you can use the sibling selector to select the label when it is a 'sibling' of a checked checkbox